src/cmd.js

```diff
@@ -15,7 +14,7 @@ const cmdTypes = {
 }
 
 export const isCmd = (object) => {
-  return object ? object[isCmdSymbol] : false
+  return object ? !!object[isCmdSymbol] : false
```

Comment:
curious about this change?

Reply:
when an object was not a cmd i just wanted it to return false instead of undefined.